python/spacewalk/spacewalk-backend.spec

```diff
@@ -252,6 +252,7 @@ Requires:       %{name}-xml-export-libs
 Requires:       cobbler
 Requires:       python3-requests
 Requires:       python3-rhnlib  >= 2.5.57
+Recommends:     susemanager-bash-completion
```

Comment:
do we really want to use recommends? 
what about suggest? at the end, spacewalk-backend can work fine without it. personally I  would  prefer to  a less strong requirement where  the user can decide to  install  this or not. 

see for example https://github.com/uyuni-project/uyuni-tools/pull/94

Reply:
AFAIK, it's pretty standard to use `recommends`/`supplements` with accompanying bash-completion packages. Your example `uyuni-tools` also uses `supplements`, not `suggests`. If we go with `suggests`, I'm worried that some users won't be even aware of this functionality.

Reply:
...but if we go with recommends the bash completion will be mandatory for all the users

Reply:
Do you mean `requires` instead of `recommends`? :)

Reply:
...sorry.I have no  idea what I was thinking about.